tutorbackup/templates/backup/build/backup/backup_services.py

```diff
@@ -43,6 +43,7 @@ def mysqldump():
     password = ENV['MYSQL_ROOT_PASSWORD']
     host = ENV['MYSQL_HOST']
     port = ENV['MYSQL_PORT']
+    single_transaction = eval(ENV['MYSQL_SINGLE_TRANSACTION'])
```

Comment:
Please don't use `eval()` with an unsanitised environment variable, use `bool(strtobool()))` instead. Something like this:

```python
try:
    from distutils.util import strtobool  # pre-3.12
except ImportError:
    from setuptools.distutils.util import strtobool  # 3.12+

[...]

flush_logs = bool(strtobool(ENV['MYSQL_SINGLE_TRANSACTION']))
```